newIDE/app/src/SceneEditor/ToolbarCommands.js

```diff
@@ -16,6 +16,8 @@ type Props = {|
   toggleWindowMask: () => void,
   toggleGrid: () => void,
   setupGrid: () => void,
+  canDuplicateObject: boolean,
+  onDuplicateObject: () => void,
```

Comment:
Not sure what to do with this, because the fonction duplicate is called in `newIDE/app/src/ObjectsList/index.js` L 1424